src/main/groovy/com/twcable/grabbit/server/batch/steps/jcrnodes/JcrNodesProcessor.groovy

```diff
@@ -84,81 +76,80 @@ class JcrNodesProcessor implements ItemProcessor<JcrNode, Node> {
         }
 
         // Skip this node because it has already been processed by its parent
-        if(isRequiredNode(jcrNode)) {
+        if(isMandatoryNode(jcrNode)) {
             return null;
         } else {
             // Build parent node
-            return buildNode(jcrNode, getRequiredChildNodes(jcrNode))
+            return buildNode(jcrNode, getMandatoryChildNodes(jcrNode))
         }
     }
 
     /**
      * Build node and "only" mandatory child nodes
-     * @param JCR Node
-     * @param Mandatory childNodes
-     * @return Node Proto node
+     * @param jcrNode
+     * @param mandatoryChildNodes
```

Comment:
Yeah, I deleted any JavaDoc statements that were trivial, and otherwise unhelpful. In my opinion, it's just pedantic noise. Like in this case : 

' @return Node Proto node'

However, I did put an @Nonnull annotation on the return to indicate callee/caller contract, as that stuff is important.